packages/bolt-connection/src/bolt/request-message.js

```diff
@@ -191,6 +191,38 @@ export default class RequestMessage {
     )
   }
 
+  /**
+   * Create a new HELLO message.
+   * @param {string} userAgent the user agent.
+   * @param {string} boltAgent the bolt agent.
+   * @param {NotificationFilter} notificationFilter the notification filter configured
+   * @param {Object} routing server side routing, set to routing context to turn on server side routing (> 4.1)
+   * @return {RequestMessage} new HELLO message.
+   */
+  static hello5x3 (userAgent, boltAgent, notificationFilter = null, routing = null) {
+    const metadata = { user_agent: userAgent, bolt_agent: boltAgent }
```

Comment:
maybe we should omit `user_agent` if it's `null` or `undefined`.

Reply:
yeah makes sense!